Convert CommonJS to ES6 syntax, correct typos and mismatched words

BaharaJr opened the pull request:
Hey chief, I looked into the words.json file and tried to make a few corrections

BaharaJr committed:
chore: add github generated folder and files

BaharaJr committed:
chore: convert function constructor into a class

I converted the function constructor into a class and changed the code from ES5 to ES6 syntax

BaharaJr committed:
fix: fix a few typos and missmatched words

Fixed some typos and corrected missmatched words

alphaolomi commented on .gitignore:
```diff
@@ -86,3 +86,6 @@ typings/
 .dynamodb/
+
+# GitHub files
+.github
```
Why remove .github files?

BaharaJr replied:
So when I clone, it comes  .github files that I would not want to push to the remote with the same files

BaharaJr replied:
> Why remove .github files?

So I figured I wasn't supposed to remove them

alphaolomi replied:
🤝

Commit:
added .github files

alphaolomi reviewed (commented):
Umetisha @BaharaJr

Commit:
delete package-lock.json

@BaharaJr Kindly use yarn [https://yarnpkg.com/lang/en/](https://yarnpkg.com/lang/en/)

alphaolomi committed:
Create yarn.lock

alphaolomi committed:
Delete test.js
